src/index.js

```diff
@@ -1 +1,61 @@
 import './index.css';
+
+// ------------ API ------------ //
+const gameId = process.env.GAME_ID;
+const baseUrl = process.env.BASE_URL;
```

Comment:
[Optional]

- [ ] Sadly the previous reviewer didn't notice this earlier so I'm putting this in the optional section. The reason why he got errors (and I did too) is because you're hiding the game ID and the base URL, I would suggest you put them as strings in the code in this case so all users can use your app without having to setup their own env file 👍

Reply:
@ha-manel  okay thanks I just wanted to apply what I learned I will change it back

Reply:
it is a good effort from you but in this case it should be public so users can use your website ☺️ great job though 👏